cpu/cortexm_common/cortexm_init.c

```diff
@@ -43,14 +47,18 @@ void cortexm_init(void)
 #if defined(CPU_ARCH_CORTEX_M3) || defined(CPU_ARCH_CORTEX_M4) || \
     defined(CPU_ARCH_CORTEX_M4F) || defined(CPU_ARCH_CORTEX_M7) || \
     (defined(CPU_ARCH_CORTEX_M0PLUS) && (__VTOR_PRESENT == 1))
+#ifndef SOFTDEVICE_PRESENT
     SCB->VTOR = (uint32_t)&_isr_vectors;
+#endif
```

Comment:
Hmm... maybe this one you can also modify it in the cpu.c of nrf52. You might assign it to the starting of the flash, where I guess the vector table is located when using the specific softdevice hex firmware.

Reply:
But then it will call twice during init
It is a bit weird isn't it ?

Reply:
It would be, but we have no choice. I prefer to initialise it twice than having this `#ifdef`s in the cortexm code.